src/main/java/com/commercetools/sync/cartdiscounts/CartDiscountSync.java

```diff
@@ -46,9 +45,8 @@
         "Failed to process cart discount draft without key.";
     private static final String CART_DISCOUNT_DRAFT_IS_NULL =
         "Failed to process null cart discount draft.";
-    private static final String FAILED_TO_RESOLVE_REFERENCES = "Failed to resolve references on "
-        + "CartDiscountDraft with key:'%s'. Reason: %s";
-
+    private static final String FAILED_TO_RUN_SYNC_DRAFT = "Failed to run sync draft process on "
```

Comment:
May be better - as it indicates than not the whole process but a single draft is affected: `Failed to process the CartDiscountDraft with key..., Reason...`